### PR TITLE
Add og meta tags

### PR DESCRIPTION
**Pull request by asdgfdf329329**
Adds social Open Graph meta tags
Read more: https://ogp.me/

### CODE REVIEW

**Review by sno2 — APPROVED** [review]

**Review by sno2 — COMMENTED** [review]
We might be better off with a `SocialMeta` component and include variables for the title, description, and other things and generate the data depending on the content of those variables.  However, this might be better achieved with a package.

File: pages/_document.tsx [hunk 1]
```diff
@@ -26,6 +26,19 @@ export default class DenoDocDocument extends Document {
...
+          <meta property="og:url" content="https://deno.land" />
+          <meta
+            property="og:title"
+            content="Deno - A secure runtime for JavaScript and TypeScript"
+          />
+          <meta
+            property="og:description"
+            content="Deno is a simple, modern and secure runtime for JavaScript and TypeScript that uses V8 and is built in Rust."
+          />
+          <meta property="og:image" content="https://deno.land/v1.png" />
```
Comment (kt3k) [inline pages/_document.tsx:40]: I think `v1.png` is for v1 announcement. I guess `https://deno.land/images/icons/apple-touch-icon-180x180.png` would be more appropriate for this usage.

**Comment by kt3k** [comment]
@asdgfdf329329 Please sign CLA from the above `CLAassistant` link

**Comment by kt3k** [comment]
Closing in favor of #1636, which also supports twitter cards. Thank you for your contribution anyway @asdgfdf329329 !

**Pull request closed without merging**